src/validations/commentValidation.ts

```diff
@@ -0,0 +1,13 @@
+import Joi from "joi";
+import { bookIdSchema } from "./bookValidation";
+
+export const commentJoiSchema = Joi.object({
+  user: Joi.string(),
```

Comment:
is user an input?
this should be gotten from the auth middleware

Reply:
it is not user input. Deleted this line